scripts/update_avalanchego_version.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+# Check if a version argument is provided
+if [ "$#" -ne 1 ]; then
+    echo "Usage: $0 <version>"
+    exit 1
+fi
+
+if ! [[ "$0" =~ scripts/update_avalanchego_version.sh ]]; then
+  echo "must be run from repository root"
+  exit 255
+fi
+
+VERSION=$1
```

Comment:
input validation for the version format of "XX.XX.XX" would be nice to have.

Reply:
I'd prefer to leave this as is. Currently this should work for a commit, release tag, or a tag that contains some `-suffix` as well.